arm_position_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - base_link_to_link1
    - link1_to_link2
    - link2_to_link3
    - link3_to_link4
    - link4_to_link5
    - link5_to_gripper_link1
  gains:
    base_link_to_link1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    link1_to_link2:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    link2_to_link3:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    link3_to_link4:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    link4_to_link5:
      p: 100
      d: 1
      i: 1
      i_clamp: 1
    link5_to_gripper_link1:
      p: 100
      d: 1
      i: 1
      i_clamp: 1